scripts/email_public_registration_contributors.py

```diff
@@ -0,0 +1,35 @@
+"""Script for sending OSF email to all users who contribute to public registrations."""
+
+import datetime
+
+from modularodm import Q
+
+from website import models
+from website.app import init_app
+from website.mails import Mail, send_mail
+
+
+def main():
+    public_registrations = models.Node.find(Q('is_registration', 'eq', True) & Q('is_public', 'eq', True))
+    contributors = []
+    for node in public_registrations:
+        contributors.extend([contrib for contrib in node.contributors if contrib not in contributors])
+
+    # FIXME: need cutoff date from Brian or Sara
+    cutoff_date = datetime.date.today() + datetime.timedelta(days=14)
```

Comment:
I would say 14 days from when the email goes out - this was the timeframe we used for the admin permissions change, I believe. @JeffSpies may also want to weigh in.